lib/stringMsg.js

```diff
@@ -28,8 +28,12 @@ function buildMsg(canIdInfo, format, data, rest = {}) {
     ...rest,
   }
 }
+function buildErrMsg(msg, input) {
+  if (input && isString(input)) return `${msg} - ${input}`
```

Comment:
Why not just `input.toString()` inside the undefined guard? or`JSON.stringify`? If the input is always a string the guard is not necessary - if it is non-string we are eager to find out what it is, aren't we?